core/CronArchive.php

```diff
@@ -1234,12 +1239,30 @@ private function request($url)
         } catch (Exception $e) {
             return $this->logNetworkError($url, $e->getMessage());
         }
+        $this->disconnectDb();
         if ($this->checkResponse($response, $url)) {
             return $response;
         }
         return false;
     }
 
+    private function disconnectDb()
+    {
+        if (empty($this->lastDbReset)) {
+            $this->lastDbReset = time();
+            return;
+        }
```

Comment:
That seems to be dead code. `$this->lastDbReset = time();` is already done in `__construct`, so the value can't ever be empty